src/renderer/remote.ts

```diff
@@ -35,8 +33,17 @@ function setCachedRemoteObject (id: number, value: any) {
   return value
 }
 
+function getContextId() {
+  const v8Util = getElectronBinding('v8_util')
+  if (v8Util) {
+    return v8Util.getHiddenValue<string>(global, 'contextId')
+  } else {
+    throw new Error('Electron with process.contextId required to support sandboxed renderers')
```

Comment:
Can we put a specific Electron version range here?

Reply:
I've put a placeholder here, which will get replaced with the actual beta when it's realeased